notebooks/02_model/ncf_deep_dive.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
Apart from testing the python files, we are also testing the notebooks, here you have an example: https://github.com/Microsoft/Recommenders/blob/staging/tests/unit/test_notebooks_python.py. It uses papermill to run the notebook. You could add the test in the file `https://github.com/Microsoft/Recommenders/blob/staging/tests/unit/test_notebooks_gpu.py`.

Apart from the unit tests for notebooks, we have smoke and integration tests that run every night. Here you have an example of smoke test in surprise https://github.com/Microsoft/Recommenders/blob/staging/tests/smoke/test_surprise_svd.py and integration tests: https://github.com/Microsoft/Recommenders/blob/staging/tests/integration/test_surprise_svd.py. As you can see, this tests run the notebook and make sure that the code retrieves reasonable metrics, the difference between smoke and integration is just the size of the database we use. Here you can find more details about the difference between tests: https://miguelgfierro.com/blog/2018/a-beginners-guide-to-python-testing/

Reply:
@miguelgfierro talked with @AaronHeee that we will develop a quick-start notebook and then add smoke/integration tests - similar to what we did with als, surprise-svd, etc. My suggestion is this can go to another PR. What do you think?